piccolo/table.py

```diff
@@ -55,6 +57,11 @@
 from piccolo.utils.sync import run_sync
 from piccolo.utils.warnings import colored_warning
 
+try:
+    from zoneinfo import ZoneInfo  # type: ignore
+except ImportError:  # pragma: no cover
+    from backports.zoneinfo import ZoneInfo  # type: ignore  # noqa: F401
+
```

Comment:
@aabmets I think you should remove this imports as it is not used anywhere in the file

Reply:
@sinisaos Done